apps/vite/src/components/UserSettings.tsx

```
@@ -1,45 +1,46 @@
+import { Fragment, useContext } from 'react';
+import { Menu, Transition } from '@headlessui/react';
+import { Cog6ToothIcon, MoonIcon, SunIcon } from '@heroicons/react/24/outline';
+import { useTranslation } from 'react-i18next';
+
+import { apiBaseUrl } from '../helper/api';
+import { classNames } from '../helper/utils';
+import { DarkModeContext } from './DarkModeProvider';
 
 function LoginWithGoogle({ darkMode }: { darkMode: boolean }) {
   return darkMode ? (
     <img src="/icons/btn_google_signin_dark_normal_web@2x.png" />
```

Biome

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)